tools/dotnet-linker/Steps/RemoveAttributesDispatcher.cs

```diff
@@ -0,0 +1,11 @@
+using Mono.Linker.Steps;
+using Xamarin.Linker;
+
+namespace Xamarin.Linker.Steps {
+	class RemoveAttributesDispatcher : SubStepsDispatcher {
```

Comment:
It might be better to name them depending on when they are called, not about what they do.

E.g. `PostSweepDispatcher`

That way the name remains meaningful (and less changes are required) if we add a new substep at that time.

The alternative is to have an incorrect/incomplete name or to make a lot more changes to add that new substep.